bigtable-hbase-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java

```diff
@@ -95,6 +96,43 @@ public void testBigtableFilter() throws IOException {
       Assert.assertTrue(CellUtil.matchingValue(result.rawCells()[0], valA));
     }
   }
+  
+  /**
+   * This test case is used to validate TimestampRangeFilter with Integer.MAX_VALUE #1552
+   * 
+   * @throws IOException
+   */
+  @Test
+  public void testTimestampRangeFilterWithMaxVal() throws IOException {
+	    // Initialize
+	    int numCols = Integer.MAX_VALUE;
```

Comment:
Were you actually able to run this?  numCols with MAX_VALUE should crash any tests.